src/DeviceInterface/InterfacePrototype.js

```diff
@@ -421,12 +421,14 @@ class InterfacePrototype {
      * When an item was selected, we then call back to the device
      * to fetch the full suite of data needed to complete the autofill
      *
-     * @param {InputTypeConfigs} config
+     * @param {import('../Form/matching').SupportedTypes} inputType
      * @param {(CreditCardObject|IdentityObject|CredentialsObject)[]} items
      * @param {CreditCardObject['id']|IdentityObject['id']|CredentialsObject['id']} id
      */
-    onSelect (config, items, id) {
+    onSelect (inputType, items, id) {
```

Comment:
I've changed this to expect the `inputType` which includes what we were actually needed, but also includes the `subtype` which we need for this new pixels.